src/server/graphql.js

```diff
@@ -47,12 +43,10 @@ const resolverMap = {
     parseLiteral(ast) {
       if (ast.kind === Kind.INT) {
         return parseInt(ast.value, 10);
-      } else {
```

Comment:
This change strikes me as strange. If we are going for a generally functional style, each if should have a corresponding else.